sdk/ai/azure-ai-inference/azure/ai/inference/_patch.py

```diff
@@ -581,43 +680,78 @@ class EmbeddingsClient(EmbeddingsClientGenerated):
      AzureKeyCredential type or a TokenCredential type. Required.
     :type credential: ~azure.core.credentials.AzureKeyCredential or
      ~azure.core.credentials.TokenCredential
+    :keyword dimensions: Optional. The number of dimensions the resulting output embeddings should
+        have.
+        Passing null causes the model to use its default value.
+        Returns a 422 error if the model doesn't support the value or parameter. Default value is
+        None.
+    :paramtype dimensions: int
+    :keyword encoding_format: Optional. The desired format for the returned embeddings.
+        Known values are:
+        "base64", "binary", "float", "int8", "ubinary", and "uint8". Default value is None.
+    :paramtype encoding_format: str or ~azure.ai.inference.models.EmbeddingEncodingFormat
+    :keyword input_type: Optional. The type of the input.
+        Returns a 422 error if the model doesn't support the value or parameter. Known values are:
```

Comment:
As we are customizing the method here, we can improve on the wording. This will raise an exception, not return a 422 error.

Reply:
oh, I removed that in the chat completions comments, but did not do it in the embeddings comments. Thanks! Will fix in next commit.